src/Core/Components/AppBar/FluentAppBarItem.razor.cs

```diff
@@ -78,6 +78,7 @@ public partial class FluentAppBarItem : FluentComponentBase, IDisposable
     public bool? Overflow { get; private set; }
 
     internal string? ClassValue => new CssBuilder("fluent-appbar-item")
+        .AddClass("fluent-appbar-item-local", when: Href == null)
```

Comment:
Just to verify: only null, not `string.IsNullOrEmpty` ?

Reply:
IsNullOrEmpy is better, yes. Will change